frontend/src/components/UserCard.tsx

```diff
@@ -45,7 +45,10 @@ const UserCard = ({
         </div>
 
         <div className="text-center">
-          <h3 className="max-w-[250px] truncate text-lg font-semibold text-gray-900 sm:text-xl dark:text-white">
+          <h3
+            className="max-w-[250px] text-lg font-semibold wrap-break-word whitespace-normal text-gray-900 sm:text-xl dark:text-white"
+            title={name}
+          >
             {name}
           </h3>
```

Comment:
Let's truncate longer names (with tooltip) so that everything is aligned properly between leaders. Also updated some styles here:


```suggestion
          <Tooltip content={name} delay={100} closeDelay={100} showArrow placement="top">
            <h3 className="max-w-[250px] truncate text-lg font-semibold text-gray-900 sm:text-xl dark:text-white">
              {name}
            </h3>
          </Tooltip>
```